src/components/Book.js

```diff
@@ -18,7 +18,7 @@ export default function Book() {
           <ul className="booklists" key={book.title}>
             <li className="lists">{book.title}</li>
             <li className="lists">{book.author}</li>
```

Comment:
- [ ] Kindly ensure you have no errors in the browser console. Fix the following error
![image](https://github.com/Sami-Ullah-Tufail/bookstore/assets/81932478/61de5ada-6409-4360-a91c-b89005488696)